src/databricks/labs/ucx/azure/credentials.py

```diff
@@ -31,21 +33,43 @@ class ServicePrincipalMigrationInfo:
 
 @dataclass
 class StorageCredentialValidationResult:
-    name: str
+    name: str | None
```

Comment:
nit: you don't need to change this, storage credential name must always be present